packages/eas-cli/src/submissions/ios/IosSubmitter.ts

```diff
@@ -35,9 +44,25 @@ class IosSubmitter extends BaseSubmitter<IosSubmissionContext, IosSubmissionOpti
       resolvedSourceOptions
     );
 
-    printSummaryIOS(submissionConfig);
-
-    await this.startSubmissionAsync(submissionConfig, this.ctx.commandFlags.verbose);
+    printSummary(
+      this.prepareSummaryData(this.options, resolvedSourceOptions),
+      'iOS Submission Summary',
+      SummaryHumanReadableKeys,
+      SummaryHumanReadableValues
+    );
+    const result = await this.startSubmissionAsync(submissionConfig, this.ctx.commandFlags.verbose);
+
+    if (result === SubmissionStatus.FINISHED) {
+      log(
+        'Your binary has been successfully uploaded to App Store Connect!\n' +
+          'It is now being processed by Apple - you will receive an e-mail when the processing finishes.\n' +
+          'It usually takes about 5-10 minutes depending on how busy Apple servers are.\n' +
+          'When it’s done, you can see your build here: ' +
+          chalk.dim.underline(
+            `https://appstoreconnect.apple.com/apps/${this.options.ascAppId}/appstore/ios`
+          )
```

Comment:
This message is pretty verbose and the format doesn't seem to match the rest of the command. 
Maybe newline it and make it more like a bullet list. Use terminal link for the link. 
Generally this message also seems too long.